src/python/bot/minimizer/js_minimizer.py

```diff
@@ -34,33 +34,58 @@ def _execute(self, data):
 
     for index, token in enumerate(testcase.tokens):
       if token == '{':
-        brace_stack.append((last_start_of_line_token, index))
+        brace_stack.append(index)
```

Comment:
Is last_start_of_line token still used at all? It would be nice to remove it if we can.

Reply:
It was being used by one of the paren hypothesis, but I found a way to remove it.